app/validators/schema_validator.rb

```diff
@@ -0,0 +1,52 @@
+require "json-schema"
+
+class SchemaValidator
+  def initialize(rendered_content_item)
+    @rendered_content_item = rendered_content_item.deep_symbolize_keys
+  end
+
+  def validate
+    return true if schema_name_exception?
+    validate_schema
+  end
+
+private
+
+  attr_reader :rendered_content_item
+
+  def validate_schema
+    JSON::Validator.validate!(schema, rendered_content_item)
+  rescue JSON::Schema::ValidationError => error
+    Airbrake.notify_or_ignore(error, parameters: {
```

Comment:
The error case is likely here (not most likely but it is likely). This actually introduces errbit as a dependency for a successful call to content store. Could we make this call async, perhaps using Airbrake's built in async mode? https://github.com/airbrake/airbrake/wiki/Using-Airbrake-asynchronously